<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>191</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QPushButton" name="ButtonNewEvent">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Add event</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonSimulate">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Simulate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonClear">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>340</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear list</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonExit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>400</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="EnterNameLine">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter a name that is no longer than 20 characters long. If the format is incorrect, the default value will be used&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="EnterChanceLine">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter a number between 1 and 1000. If the input is incorrect, the default value will be used&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonCreate">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
    <widget class="QLabel" name="EnterNameLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter name</string>
     </property>
    </widget>
    <widget class="QLabel" name="EnterChanceLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter chance(integer)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonNormalizeShown">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Normalize events</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonSimulateShown">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Simulate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonClearShown">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear list</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonExitShown">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="EnterNumberOfSimulationLine">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter a number between 1 and 1000. If the input is incorrect, the default value will be used&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="EnterNumberOfSimulationLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>190</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter number of simulation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonExitShown_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>350</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonClearShown_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>290</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear list</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_Normalise">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Normalize events</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonDoIt1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>240</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Do it</string>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="ListResult">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>110</y>
      <width>211</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="ListEvents">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>271</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Events">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>List of events</string>
    </property>
   </widget>
   <widget class="QLabel" name="History">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>90</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>History</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EnterSeed">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;As a seed, a number between 1 and 1000 is used. If the input is uncorrect, the seed will be the previous seed&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="SeedLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Seed</string>
    </property>
   </widget>
   <widget class="QListWidget" name="ListStatistic">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>110</y>
      <width>271</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="StatsLabel">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>90</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Statistic</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1160</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
